src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -472,87 +472,87 @@
                 <source>This file is not a valid video.</source>
-                <target state="needs-review-translation">Diese Datei ist kein gültiges Video.</target>
+                <target>Diese Datei ist kein gültiges Video.</target>
             </trans-unit>
             <trans-unit id="123">
                 <source>The size of the video could not be detected.</source>
-                <target state="needs-review-translation">Die Größe des Videos konnte nicht ermittelt werden.</target>
+                <target>Die Größe des Videos konnte nicht ermittelt werden.</target>
             </trans-unit>
             <trans-unit id="124">
                 <source>The video width is too big ({{ width }}px). Allowed maximum width is {{ max_width }}px.</source>
-                <target state="needs-review-translation">Die Videobreite ist zu groß ({{ width }}px). Zulässige maximale Breite ist {{ max_width }}px.</target>
+                <target>Das Video ist zu breit ({{ width }}px). Die zulässige maximale Breite ist {{ max_width }}px.</target>
             </trans-unit>
             <trans-unit id="125">
                 <source>The video width is too small ({{ width }}px). Minimum width expected is {{ min_width }}px.</source>
-                <target state="needs-review-translation">Die Videobreite ist zu klein ({{ width }}px). Erwartete Mindestbreite ist {{ min_width }}px.</target>
+                <target>Das Video ist nicht breit genug ({{ width }}px). Die erwartete Mindestbreite ist {{ min_width }}px.</target>
             </trans-unit>
             <trans-unit id="126">
                 <source>The video height is too big ({{ height }}px). Allowed maximum height is {{ max_height }}px.</source>
-                <target state="needs-review-translation">Die Videohöhe ist zu groß ({{ height }}px). Zulässige maximale Höhe ist {{ max_height }}px.</target>
+                <target>Die Videohöhe ist zu groß ({{ height }}px). Zulässige maximale Höhe ist {{ max_height }}px.</target>
             </trans-unit>
             <trans-unit id="127">
                 <source>The video height is too small ({{ height }}px). Minimum height expected is {{ min_height }}px.</source>
-                <target state="needs-review-translation">Die Videohöhe ist zu klein ({{ height }}px). Erwartete Mindesthöhe ist {{ min_height }}px.</target>
+                <target>Die Videohöhe ist zu klein ({{ height }}px). Erwartete Mindesthöhe ist {{ min_height }}px.</target>
             </trans-unit>
             <trans-unit id="128">
                 <source>The video has too few pixels ({{ pixels }} pixels). Minimum amount expected is {{ min_pixels }} pixels.</source>
-                <target state="needs-review-translation">Das Video hat zu wenige Pixel ({{ pixels }}). Erwartete Mindestmenge ist {{ min_pixels }}.</target>
+                <target>Das Video hat zu wenige Pixel ({{ pixels }}). Erwartete Mindestanzahl ist {{ min_pixels }}.</target>
```

Comment:
What does "too few pixels" even mean? 🙈

Reply:
h x w = amount :shrug:

Reply:
Yeah no clue either. 😄 Do we talk about the resolution of the video here?

Reply:
the number of pixels in frames - we have the same option for images

Reply:
Can we reword this a bit to make that more clear? What about using the word resolution instead?

Reply:
I wouldn't. I'm not sure what this option is useful for but I'd stick to the original text (we won't have this conversation in all languages :) )